moped-database/migrations/1703786941217_modify_files_user_id_constraint/up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE moped_project_files DROP CONSTRAINT "moped_project_files_created_by_fkey";
+
+ALTER TABLE moped_project_files ADD CONSTRAINT "moped_project_files_created_by_fkey" FOREIGN KEY ("created_by") REFERENCES "public"."moped_users"("user_id") ON DELETE CASCADE ON UPDATE CASCADE;

```

Comment:
ah—we've never had to cascade a user ID change!

i don't feel strongly, but i am slightly worried about cascading deletes from the user table. i think it might be unintuitive that a user deletion causes their file data to delete, but no other user-related tables cascade deletes afaik.

Reply:
cool! yeah, i was a little weary of the delete part too. I'll update it to on work on update. 🙏